intro_to_ada.rst

```diff
@@ -152,15 +145,17 @@ unsurprising result.
 
 There are several note worthy things in the above program:
 
--  A procedure is like a C/C++ function returning ``void``. We'll see
-   later how to declare proper functions.
-
-.. gusthoff: This paragraph does not really explain what a procedure actually is.
+-  A subprogram in Ada can be either a procedure or a function. A
```

Comment:
AI for me: Go in a bit more detail, put a link to wiki/similar about functions/subprograms in PLs